src/core/IO/Socket/INET.pm

```diff
@@ -89,6 +90,7 @@ my class IO::Socket::INET does IO::Socket {
                $family == PIO::PF_INET
             || $family == PIO::PF_INET6
         } = PIO::PF_INET,
+        Str :$source-address is copy = "".Str,
```

Comment:
`:$source-address` should probably be indented a couple spaces to line it up with the rest of the parameters. Also, you shouldn't need the `.Str` on the `""`.